sudokuru/Changelog.json

```diff
@@ -1,7 +1,17 @@
 ﻿[
   {
-    "version": "1.28.1",
+    "version": "1.28.2",
     "date": "#{date}#",
+    "summary": "When deleting statistics learned lessons would temporarily display nothing on profile page instead of None.",
+    "bug fixes": [
+      "After deleting statistics learned lessons immediately display as None on profile page."
```

Comment:
I think there should be a comma here after statistics
<img width="328" alt="image" src="https://github.com/user-attachments/assets/df7de840-fb3f-42df-8c6b-0ca5619dc7c8" />

Reply:
or maybe rephrasing would be good:
<img width="329" alt="image" src="https://github.com/user-attachments/assets/389dbf35-09c4-4bb3-823b-e60dc98b8aa7" />